packages/example/.gitignore

```diff
@@ -0,0 +1,11 @@
+node_modules/**/*
+.expo/*
+npm-debug.*
+*.jks
+*.p8
+*.p12
+*.key
+*.mobileprovision
+*.orig.*
+web-build/
+web-report/
```

Comment:
these 2 are featured in both gitignores, check which one you need. I think I'd rather go with a single gitignore in the root